fabric-content-registries-v1/src/main/java/net/fabricmc/fabric/impl/content/registries/FuelRegistryImpl.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.impl.content.registries;
+
+import java.util.Map;
+
+import com.google.common.collect.Maps;
+
+import net.minecraft.item.Item;
+
+import net.fabricmc.fabric.api.content.registry.v1.FuelRegistry;
+
+public class FuelRegistryImpl implements FuelRegistry {
+
+	public static final Map<Item, Integer> FUELS = Maps.newHashMap();
```

Comment:
wasn't this supposed to be an object, why static?

Reply:
Done, made it an instance field